tahowallet/taho.xyz
# Updated new Twitter link

### Notes
* Updated the Twitter links in homepage footer and goodbye pages from `tallycash` to our new handle `TallyHoOfficial`

### Test
* Click on Twitter icon in homepage footer and confirm it directs to `https://twitter.com/TallyHoOfficial`
* Click on Twitter icon on goodbye page and confirm it directs to `https://twitter.com/TallyHoOfficial`
